### DISCUSSION TITLE
the feature map always have a box

### DISCUSSION THREAD

**Discussion by Phyonna** [open-edge-platform/anomalib#1999] [Q&A] [answered]
as you can see,all the predict result (including good samples and bad samples)have a anomaly box on the image(the yellow box).I guess that might caused by the center_crop,but after I close center_crop,there still have a anomaly box  . why will this happen?
![004](https://github.com/openvinotoolkit/anomalib/assets/102140144/26902ddf-e34c-40a2-8688-5366a4cf762b)
![018](https://github.com/openvinotoolkit/anomalib/assets/102140144/f7aca0ae-4ca7-40db-93ea-a4827696a468)
![004](https://github.com/openvinotoolkit/anomalib/assets/102140144/5c555d3d-10ba-4830-9991-24ebb4b8d315)

**Comment by abc-125** [comment, marked as answer]
Hi, if you are using PatchCore, it is normal. You can take a look at [sample results](https://github.com/openvinotoolkit/anomalib/tree/main/src/anomalib/models/image/patchcore#sample-results), there is a similar box at every image. I assume it happens because of how feature extraction and especially feature reduction work in this model.

**Reply by Phyonna** [reply]
thank you! I got it.

**Comment by TurboJonte** [comment]
I have the same fenomena using EfficientAD, I ran the training for 100 epochs on 23 images, the result is really bad.
![240229_145529_602_Band Fail_Bds_D28310996_Sp38_SLICE4_CAM4_0](https://github.com/openvinotoolkit/anomalib/assets/10991136/70fb4bab-9e76-4b90-b9e6-e977651e18d7)

**Reply by abc-125** [reply]
It is a different type of box :) EfficientAD has a default setting `padding: false`, which gives you such results by using only the area in this green square to detect anomalies. 

For such a low number of training images (23), I strongly recommend using PatchCore instead. But if you want to use EfficientAD, please take a look at this [pull request](https://github.com/openvinotoolkit/anomalib/pull/2015), which should fix some performance problems. And it is better to run it with 70000 iterations, according to the paper, which is 3044 epochs in your case.

**Reply by TurboJonte** [reply]
@abc-125 Thank you for your reply! I did not know about the padding setting.
I've managed to get some more images, with some augmentation I now have approx 400.
I've seen the pull request and it's interesting for sure! I've installed anomalib from source but this request is still not pulled it seems.
I'm at the moment in the progress of a 1000 epoch training with the 23 images, we'll se how that comes out.
